src/customize-pop/online-accounts.md

```diff
@@ -0,0 +1,44 @@
+# Online Accounts
+
+Settings to connect online accounts are located in `Settings` ➞ `Online Accounts`. Online Accounts settings provide single sign-on for online providers. Connected accounts can directly access core services like Mail, Calendar, Contacts, Maps, Photos, Files, and Music, without requiring sign-on through a web browser.
```

Comment:
"Online Accounts settings" reads weird (it would more naturally be "Online account settings," but I know you're trying to use the proper name of the section.) Consider using the more generic phrase instead of the actual name, or avoid the issue by saying "These settings..." instead.

I would recommend just saying "services" instead of "core services"; referring to them as "core" seems kind of arbitrary, and some of these are definitely not fundamental/basic things.

Reply:
Updated